homeassistant/helpers/update_coordinator.py

```diff
@@ -87,7 +87,10 @@ def __init__(
         )
 
         self._listeners: dict[CALLBACK_TYPE, tuple[CALLBACK_TYPE, object | None]] = {}
-        self._job = HassJob(self._handle_refresh_interval)
+        job_name = f"DataUpdateCoordinator {name}"
```

Comment:
Should we include `type(self).__name__` ? Data update coordinators are often also extended using inheritance.

Reply:
👍 

f0a4c5a1adcd0852b1130ed043a62ad9de99d39a